src/react/hooks/use-presence.ts

```diff
@@ -1,4 +1,4 @@
-import { type ErrorInfo } from 'ably';
+import { ChannelStateChange, type ErrorInfo } from 'ably';
```

Comment:
Everywhere else we do `import * as Ably from 'ably'` so it's clear what is coming from ably-js and what isn't. Can we do the same here?